app/helpers/search_helper.rb

```diff
@@ -2,41 +2,42 @@ module SearchHelper
 
   # modified from mislav-will_paginate-2.3.2/lib/will_paginate/view_helpers.rb
   def search_header(collection, search, item_name, parent=nil)
-    header = ""
+    header = []
     if !collection.respond_to?(:total_pages)
-      header = ts "Recent #{item_name.pluralize}"
+      header << ts("Recent #{item_name.pluralize}")
     elsif collection.total_pages < 2
-      header = pluralize(collection.size, item_name)
+      header << pluralize(collection.size, item_name)
     else
       total_entries = collection.total_entries
       total_entries = collection.unlimited_total_entries if collection.respond_to?(:unlimited_total_entries)
-      header = %{ %d - %d of %d }% [
+      header << %{ %d - %d of %d }% [
```

Comment:
Uh, I have no idea what's going on here. Might research later, since "baffling" is not a very maintainable state for code.

Reply:
Thinking out loud:

What do we gain here by quoting a string with the percent notation (`%{}`)? The fact that we then use the percent notation for string formatting (`%d`) and the operator `%`, for an amazing total of 3 different meanings for the same cursed symbol in one line of code, makes it a contender for obfuscated Ruby.

The cop isn't helping much by plonking all the "offenses" at once, even though the final suggestion seems to be "get rid of one of these `%`",  but at least it got me to understand the many meanings of `%`. 🤔

Reply:
Do we need `format`, specifically, for its formatting properties? I assume we're using `%d` because we want to get something out of the number format. How would we express that as a template token like `%{foo}`? Is it `%d{foo}`?

Is this format compatible with `ts`, so we can translate the entire header?